packages/vite-plugin-svelte/src/utils/load-svelte-config.ts

```diff
@@ -38,7 +39,10 @@ export async function loadSvelteConfig(
 		// try to use dynamic import for svelte.config.js first
 		if (configFile.endsWith('.js') || configFile.endsWith('.mjs')) {
 			try {
-				const result = await dynamicImportDefault(pathToFileURL(configFile).href);
+				const result = await dynamicImportDefault(
+					pathToFileURL(configFile).href,
+					fs.statSync(configFile).mtime
```

Comment:
```suggestion
					fs.statSync(configFile).mtimeMs
```

maybe? Just using `mtime` will get a `Date` instance I believe, which will then be formatted via whatever localization you happen to get with `.toString()`, which is going to contain spaces and other characters that should be URL encoded.

`mtimeMs`, on the other hand, is a decimal number. Might not matter, but it seems a bit tidier.